SourceCode/GPS/IO/FileIOUtils.cs

```diff
@@ -0,0 +1,72 @@
+﻿using System;
+using System.Collections.Generic;
+using System.Globalization;
+using System.IO;
+using AgOpenGPS.Core.Models;
+
+namespace AgOpenGPS.IO
+{
+    public static class FileIoUtils
+    {
+        // ---- Formatting helper ----
+        public static string FormatDouble(double value, int decimals)
+        {
+            return Math.Round(value, decimals).ToString(CultureInfo.InvariantCulture);
+        }
+
+        // ---- IO helpers ----
+        public static void EnsureDir(string fieldDirectory)
+        {
+            if (string.IsNullOrWhiteSpace(fieldDirectory))
+                throw new ArgumentNullException(nameof(fieldDirectory));
+            Directory.CreateDirectory(fieldDirectory);
+        }
+
+        public static void SkipBlanks(string[] lines, ref int idx)
+        {
+            while (idx < lines.Length && string.IsNullOrWhiteSpace(lines[idx])) idx++;
+        }
+
+        // ---- Parsing helpers ----
+        public static bool TryParseEN(string line, out double e, out double n)
```

Comment:
Do not abbreviate:
```suggestion
        public static bool TryParseEastingNorthing(string line, out double easting, out double northing)
```
or even better:
```suggestion
        public static bool TryParseVec2(string line, out vec2 result)
```